Src/DependencyCollector/Shared/Implementation/ProfilerSqlCommandProcessing.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace Microsoft.ApplicationInsights.DependencyCollector.Implementation
+{
+    using System.Data;
+    using System.Data.SqlClient;
+    using Microsoft.ApplicationInsights.DependencyCollector.Implementation.Operation;
+    using Microsoft.ApplicationInsights.Extensibility;
+
+    /// <summary>
+    /// Concrete class with all processing logic to generate RDD data from the calls backs
```

Comment:
updated comments

---
In reply to: [98076447](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/298#discussion_r98076447) [](ancestors = 98076447)